v2_1/ws/rest/src/sdmx-rest.yaml

```diff
@@ -56,10 +63,11 @@ paths:
         '200':
           $ref: '#/components/responses/200'
 
-
   /availability/{context}/{agencyID}/{resourceID}/{version}/{key}/{componentID}:
     get:
-      summary: 'Data availability query'
+      summary: 'Data availability queries'
+      tags: 
+        - Data queries
```

Comment:
Shouldn't the tag be `Structure queries` instead of `Data queries`, since the response is a (dynamic) structure artefact - it's about information on data, but not data itself?

Reply:
Thanks, @dosse. Yes, I was not sure either, to be honest. My point was that data availability and data validity were more about data than structural metadata, even though they both capture their content as constraints... So, it's more about the intent of the consumer than the type of artefacts contained in the message. Do you think that's wrong?

Reply:
Further thinking, data availability queries are not about data. There are about retrieving those structural artefacts for which data exists. We are looking for structures that have a certain characteristic. It happens to be the characteristic of data availability but it could have been another characteristic, e,g, getting those structures for which ref metadata exists...   
Naturally, what you receive in return are structures and not data or ref metadata...